scripts/reset-user-testing.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -euo pipefail
+
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+
+# shellcheck disable=SC1090
+source "${SCRIPT_DIR}/common.sh"
+
+check_aws_account_used prod
```

Comment:
It would be convenient to have a variable that can be easy to override to specify the environment:

```
TARGET_AWS_ACCOUNT=${TARGET_AWS_ACCOUNT:-prod}
``` 
and then use `$TARGET_AWS_ACCOUNT` instead of hardcoded prod everywhere.

The same for the `--no-email` option, that was added in a previous commit but not used. You could use a variable like:

```
"${SCRIPT_DIR}"/rotate-user-password.sh -e prod -u holly.challenger+1@digital.cabinet-office.gov.uk ${NO_EMAIL:+--no-email}
```

So if `$NO_EMAIL` is set, it will add the variable there.